<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pro_sel_dlg</class>
 <widget class="QDialog" name="pro_sel_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>計算機科學與技術研習用時：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>閲讀調研（文史哲）用時+:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rpro_sel_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_sel_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>寫作用時+:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>藝術學習（電影繪畫）用時+:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wpro_sel_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_sel_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>備注:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alpro_sel_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ctlpro_sel_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sel_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>查 詢</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
